### PR TITLE
📦 switching to dpy 2.0, new help command, fixes

### PR DESCRIPTION
**Pull request by Kile**
This pr does not add a lot but fixes numerous bugs, cleans up messy code and starts using discord.py 2.0 using it's functionalities

This PR adds:
+ A generic paginator in `paginator.py` using buttons
+ A new help command, making it more dynamic and hopefully increasing user friendliness
+ More error handler messages

This PR changes:
+ rename `functions.py` -> `checks.py`
+ `.gitignore` now includes `**/env` which prevents a virtual environment to be committed
+ There have been changes in the README file appropriate to point 2
+ `web-scraping.py` has now fully switched to asyncronous requests
+ `k!profile` now includes the discord certified moderator badge as well as the nitro badge (if possible)
+ All owner restricted commands now use the `@commands.is_owner()` decorator instead of an `if` statement inside of the command
+ The prefix command has been moved from `prefix.py` (which is now deleted) to `moderation.py`
+ All breaking changes from dpy 1.7 -> 2.0 have been changed
+ The not used commands in `__init__.py` have been commented out

This PR fixes:
+ A very ugly message if the id passed to `k!profile` was invalid
+ No response when a certain error occured on all pxlapi commands
+ An issue where the path to `font.ttf` had to be changed based on the machine it was run on
+ An issue where you were unable to use certain commands in dms with Killua
+ An issue with `k!uwufy` where it would add a line with just a `-`

**All these changes and features have been tested**

How commands have been changed for the help command:
```diff
- @commands.command()
+ @commands.command(extras={"category": Category.FUN}, usage="img <query>")
async def img(self, ctx, *, query:str):
-   #h Search for any image you want
-   #u img <query>
+   """Search for any image you want"""
    ...
```

TLDR; Killua switched to d.py 2.0 and uses prettier menus, plus is cleaner internally. It has a more appealing help command and had some bugs fixed.

### CODE REVIEW

**Comment by Kile** [comment]
Good enough
